### ISSUE TITLE
The |- (go to start of) Event button is not working properly

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#174]
If there is an event with an onset, when the event is selected, that should go to the start frame of the event, not of the onset.  It doesn't. 

<img width="538" alt="screen shot 2017-08-21 at 10 32 16 pm" src="https://user-images.githubusercontent.com/13629122/29546189-cbbfbfac-86c0-11e7-87c8-6a07f352d7b4.png">

The 'go to end of event' button works ok with respect to the offset.  That is, if the event is selected, the "go to end" of event button goes correctly to the end of the event (not to the end of the offset).

HOWEVER, if the offset is selected, the go to beginning of event should go to the beginning of the offset, but it goes instead to the beginning of the entity

<img width="673" alt="screen shot 2017-08-21 at 10 35 23 pm" src="https://user-images.githubusercontent.com/13629122/29546232-1487c3a6-86c1-11e7-8bb7-c2452492fb7c.png">

When there is both an onset and an event, when the offset is selected, the go to beginning of event goes to the beginning of the onset (rather than the beginning of the offset)

<img width="588" alt="screen shot 2017-08-21 at 10 36 23 pm" src="https://user-images.githubusercontent.com/13629122/29546255-428d223c-86c1-11e7-88fe-1849f911b94f.png">

In other words, the buttons associated with go to beginning of "event" have the effect of going to the beginning of the entity, whereas the go to end of event button works properly.

The "play event" function seems to work properly, except for sometimes overshooting by one frame the endpoint:

<img width="221" alt="screen shot 2017-08-21 at 10 45 28 pm" src="https://user-images.githubusercontent.com/13629122/29546454-98320544-86c2-11e7-8211-8df4ed3e0931.png">

**Comment by gregorydimitriadis** [comment]
Closing as fixed as per Carol on 2017-09-06
